### DISCUSSION TITLE
Variables *ship* not populated

### DISCUSSION THREAD

**Discussion by jeromedaumand** [EDCD/EDDI#2508] [EDDI VoiceAttack Q&A] [answered]
Hi every body,
first of all, i realy would like to thanks you for the great job :)

Now my problem is that i don't understand why the ship variables are not updated by EDDI.
i have tried to call them in VA via
{TXT:Ship ...} or {TXT:EDDI ship}
always return not set value

tested even in the default EDDI profile VAP
the only variables working correctly are :
- Commander
- station
- system

EDDI version is 4.0.2
VA is 1.10.3 64b steam edition

Plugin are activated and loaded correctly
![image](https://user-images.githubusercontent.com/56674265/216821112-06c79804-9363-45bf-99e7-829cebb055d8.png)

i attached the last log from EDDI and Elite's journal

[eddi.log](https://github.com/EDCD/EDDI/files/10611126/eddi.log)
The EDDI log show some errors i don't understand.
[Journal.2023-02-05T124110.01 - Copie.log](https://github.com/EDCD/EDDI/files/10611031/Journal.2023-02-05T124110.01.-.Copie.log)

Thanks in advance for any help and if something is missing, i would provided it

**Comment by jeromedaumand** [comment]
and some little precision.
I'm french and playing on horizon, not odyssey :)

**Comment by Tkael** [comment, marked as answer]
Hi there. This is a bug that was identified in 4.0.2 and which should be fixed in the next release. Ref. #2482

**Reply by jeromedaumand** [reply]
hi and thank you very much and sorry 
visibly, i haven't serched enough :'(

**Comment by Vince1024** [comment]
Hi,

When do you think this could be fixed, in other words when do you think the new version will be released?
This problem is very blocking for me to create new macros in VA because these values ​​are not always filled in.

Thank you for your excellent work.
Vincent.

**Reply by Tkael** [reply]
Hopefully soon now. I'm doing private testing and trying to iron out any obvious bugs ahead of a public release but I think we're in a pretty good place right now.
